.bazelversion

```diff
@@ -1 +1 @@
-6.1.1
```

Comment:
This is a bug for `rules_bazel_integration_test` on Windows where if an empty line exists at the end of file, a `\r` will get added to the end of the version and ruin the url format. Removing the newline will make it correctly parse the version.

Reply:
haha, ouch, we should fix there but sounds good for now